### PR TITLE
Character movement PoC with by-pixel checks

### PR DESCRIPTION
**Pull request by barnabasbartha**

### CODE REVIEW

File: src/main/client/scene/game-scene.ts [hunk 1]
```diff
@@ -17,38 +21,108 @@ export class GameScene extends Scene {
       });
    }
 
+   private sceneWidth: number;
+   private sceneWidth2: number;
```
Comment (dolanmiu) [inline src/main/client/scene/game-scene.ts:25]: why is there is a second screen width

Comment (dolanmiu) [reply src/main/client/scene/game-scene.ts:25]: oh ok, its half of it

File: webpack.config.ts [hunk 2]
```diff
@@ -1,26 +1,26 @@
-import * as path from 'path';
-import OptimizeCssAssetsPlugin from 'optimize-css-assets-webpack-plugin';
-import MiniCssExtractPlugin from 'mini-css-extract-plugin';
-import HtmlWebPackPlugin from 'html-webpack-plugin';
-import { CleanWebpackPlugin } from 'clean-webpack-plugin';
-import { Configuration, HotModuleReplacementPlugin } from 'webpack';
+import * as path from "path";
```
Comment (dolanmiu) [inline webpack.config.ts]: can you reformat this

File: src/main/client/scene/game-scene.ts (hunk 1 above)
Comment (barnabasbartha) [reply src/main/client/scene/game-scene.ts:25]: Yes we can clean up sum clean-code work later, just wrote a bunch of poc here :D

File: webpack.config.ts (hunk 2 above)
Comment (barnabasbartha) [reply webpack.config.ts]: Oh prettier runs only in src, good catch.

**Review by dolanmiu — APPROVED** [review]

**Pull request merged**